content/blog/the-importance-of-ubiquitous-language/index.md

```diff
@@ -0,0 +1,188 @@
+---
+title: The Importance of Ubiquitous Language
+description: Exploration of Ubiquitous Language - What it means, how it helps, and how it is discovered and maintained.
+date: '2023-01-12'
+slug: ubiquitous-language
+tags:
+- team
+- architecture
+- "domain driven design"
+team:
+- Greg Ball
+---
+
+Good communication is one of the hardest things to be effective at. In relationships, in business, in teams, communication has and continues to be at the top of the list when it comes to what makes them good or bad.  Core to communication in general, is the ability to speak the same language.  On a software team, this can be accomplished by speaking a language that is **_ubiquitous_**.  
+
+> <span style="font-size: 50px; line-height: 50px">Ubiquitous</span>
+> <br/>
+> &nbsp;&nbsp;[yoo-**bik**-wi-_tuhs_] *adjective*
+> 
+> &nbsp;&nbsp;&nbsp;&nbsp;present, appearing, or **found everywhere**
```

Comment:
Please remove this HTML and use available markdown styling, such as `>` for quote, `_x_` for emphasis, `*xyz*` for bold, etc.

Reply:
Removed HTML